features/backend/product_archetypes.feature

```diff
@@ -76,6 +76,7 @@ Feature: Product archetypes
         Then I should be on the product archetype index page
         And I should see "Archetype has been successfully created."
 
+    @javascript
     Scenario: Creating a product by building archetype
```

Comment:
Let's leave it like this for now, we are reworking Behat suite so let's avoid adding scenarios with old structure if not absolutely necessary.